### PR TITLE
Replace remaining fontawesome dropdown icons with SVG

### PR DESCRIPTION
**Pull request by silverwind**
- Replace leftover dropdown triangles with SVG
- Replace remove icon with SVG and add styling for it:

Screenshot 2023-05-01 at 00 40 05
Screenshot 2023-05-01 at 00 46 56

### CODE REVIEW

File: templates/repo/graph.tmpl [hunk 1]
```diff
@@ -8,7 +8,6 @@
 				<div class="ui icon buttons tiny color-buttons">
 					<div class="ui multiple selection search dropdown" id="flow-select-refs-dropdown">
 						<input type="hidden" name="flow">
-						<i class="dropdown icon"></i>
```
Comment (delvh) [inline templates/repo/graph.tmpl:11]: What happened to this icon?

File: templates/repo/issue/fields/dropdown.tmpl [hunk 2]
```diff
@@ -3,7 +3,8 @@
 	{{/* FIXME: required validation */}}
 	<div class="ui fluid selection dropdown {{if .item.Attributes.multiple}}multiple clearable{{end}}">
 		<input type="hidden" name="form-field-{{.item.ID}}" value="0">
-		<i class="dropdown icon"></i>
+		{{svg "octicon-triangle-down" 14 "dropdown icon"}}
+		{{svg "octicon-x" 14 "remove icon"}}
```
Comment (delvh) [inline templates/repo/issue/fields/dropdown.tmpl:7]: And what happened here?

File: templates/repo/graph.tmpl (hunk 1 above)
Comment (silverwind) [reply templates/repo/graph.tmpl:11]: It was previously hidden via CSS so was never visible. The field still works without it as expected (git graph branch selector).

File: templates/repo/issue/fields/dropdown.tmpl (hunk 2 above)
Comment (silverwind) [reply templates/repo/issue/fields/dropdown.tmpl:7]: Fomantic automatically inserts `<i class="remove icon">` via JS when at least 1 item is present and the icon is absent in HTML. With this present, fomantic will no longer try to add it and use this icon instead.

**Review by delvh — APPROVED** [review]

File: templates/repo/issue/fields/dropdown.tmpl (hunk 2 above)
Comment (silverwind) [reply templates/repo/issue/fields/dropdown.tmpl:7]: Fomantic JS here:

https://github.com/go-gitea/gitea/blob/fe2fe852634c0f2ee4f7360fc424f2675e198cdf/web_src/fomantic/build/semantic.js#L3180-L3186

**Comment by silverwind** [comment]
One more related tweak, active css added for label:

Screenshot 2023-05-01 at 01 16 16

**Comment by wxiaoguang** [comment]
I think we should put these "best practices" into our devtmpl, the future contributors could simply copy & paste the code. Otherwise more `<i icon>` would come in the future.

**Comment by silverwind** [comment]
> I think we should put these "best practices" into our devtest/dropdown.tmpl, the future contributors could simply copy & paste the code, and we can tell them "please use the example in devtest/dropdown.tmpl". Otherwise more `<i icon>` would come in the future.

I plan to remove FontAwesome next, so `<i>` will no longer work then. Just need to confirm there's no more use coming from Fomantic I guess.

**Review by yardenshoham — APPROVED** [review]

**Comment by wxiaoguang** [comment]
> > I think we should put these "best practices" into our devtest/dropdown.tmpl, the future contributors could simply copy & paste the code, and we can tell them "please use the example in devtest/dropdown.tmpl". Otherwise more `<i icon>` would come in the future.
> 
> I plan to remove FontAwesome next, so `<i>` will no longer work then. Just need to confirm there's no more use coming from Fomantic I guess.

I  think having examples in our devtest templates still helps developers , because when using Fomantic components, our code is different from the official one. If some contributors copy the Fomantic official code into Gitea, they will be surprised.

**Comment by wxiaoguang** [comment]
For example, everyone could have the question:

```
<div "ui dropdown">
	{{svg "octicon-triangle-down" 14 "dropdown icon"}}
	{{svg "octicon-x" 14 "remove icon"}}
</div>
```

Why there are two SVG elements ..... no document mentions that.

**Comment by silverwind** [comment]
I think most people just copy existing HTML when creating new ones. They don't really care how it works under the hood or don't read documentation anyways. Seen it too many times where legacy stuff like inline styles are copied into new PRs.

**Comment by wxiaoguang** [comment]
> I think most people just copy existing HTML when creating new ones. They don't really care how it works under the hood or don't read documentation anyways. Seen it too many times where legacy stuff like inline styles are copied into new PRs.

That's just because we do not have the best practice guideline.

And the examples could also help reviewing, if some people write non-ideal code, we can just tell them: see the devtest examples.

**Comment by silverwind** [comment]
Devtest examples are good, but only if they are kept up to date. I need to look into those.

**Pull request merged**